msal/application.py

```diff
@@ -823,6 +836,31 @@ def acquire_token_by_refresh_token(self, refresh_token, scopes):
             on_removing_rt=lambda rt_item: None,  # No OP
             )
 
+    def acquire_token_interactive(self,
+            scopes,  # type: list[str]
+            # additional_scope=None,  # type: Optional[list]
+            login_hint=None,  # type: Optional[str]
+            port=None,
+            prompt=None,
+            domain_hint=None,  # type: Optional[str]
+            claims_challenge=None,
+            **kwargs):
```

Comment:
Remove this param, for now.

Reply:
Given that we do not handle `**kwargs` in the implementation, I mean to say let's remove this `**kwargs` from this method's signature.

Reply:
Yes, I have already removed it in the latest commit